### DISCUSSION TITLE
How to import/embed a Cetz/Typst file in another Typst file?

### DISCUSSION THREAD

**Discussion by tapyu** [cetz-package/cetz#590] [Q&A] [answered]
I have `a.typ` which is a (relatively long) Cetz image, and I want to embed it in `b.typ`. Is there something like the LaTeX `standalone` package that allows me to do so?

**Comment by tapyu** [comment]
For those who were used to `standalone` in LaTeX: Just create a `.svg` by running `typst compile a.typ a.svg`, and then import it in `b.typ` as any other ordinary image.

**Comment by fenjalien** [comment, marked as answer]
You could use the `include` statement. Its like importing but it places the content of the file into your document instead of importing the variables. https://typst.app/docs/reference/scripting#modules

**Reply by tapyu** [reply]
That is insane how it works like a charm. For those who want to put Cetz into a figure, you can simply do something like that

```typ
#figure(
canvas(length: 1.5cm, { ...
...
caption: "My captions",
)
```

And then you just call it in `main.typ` with `#include`.
